Datadog.Serverless.Compat/DatadogServerlessCompat.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using Microsoft.Extensions.Logging;
+
+namespace Datadog.Serverless.Compat
+{
+    public enum CloudEnvironment
+    {
+        AzureFunction,
+        AzureSpringApp,
+        GoogleCloudRunFunction1stGen,
+        Unknown
+    }
+
+    public static class DatadogServerlessCompat
+    {
+        private static readonly string OS = RuntimeInformation.OSDescription.ToLower();
+        private static readonly string? BinaryPath = Environment.GetEnvironmentVariable(
+            "DD_SERVERLESS_COMPAT_PATH"
+        );
```

Comment:
`BinaryPath` is only used in `GetBinaryPath()`, so we can make this a local variable in that method instead of a static field.

Reply:
Updated here: https://github.com/DataDog/datadog-serverless-compat-dotnet/commit/4bf976683223452d655a99249385cd93cf015763